### ISSUE TITLE
fairseq-train: error: unrecognized arguments   when run speech_to_text example

### ISSUE THREAD

**Issue by zhongkang-ustc** [facebookresearch/fairseq#3997]
**train:**
fairseq-train /home4T_1/kzhong/study/s2t/data \
  --config-yaml config_st.yaml \
  --train-subset train_de_st,train_es_st \
  --valid-subset dev_de_st,dev_es_st \
  --lang-pairs en-de,en-es \
  --save-dir /home4T_1/kzhong/study/s2t/output/multi/ad_ft --num-workers 4 --max-tokens 40000 --max-update 100000 \
  --task speech_to_text --criterion label_smoothed_cross_entropy --label-smoothing 0.1 --report-accuracy \
  --arch s2t_transformer_m --ignore-prefix-size 1 --optimizer adam --lr 2e-3 --lr-scheduler inverse_sqrt \
  --warmup-updates 10000 --clip-norm 10.0 --seed 1 --update-freq 8 \
  --find-unused-parameters \
  --homogeneous-batch \
  --adapter-enc-dim 256 \
  --adapter-enc-type 'per_lang' \
  --adapter-dec-dim 256 \
  --adapter-dec-type 'per_lang' \
  --finetune-enc-modules adapter \
  --finetune-dec-modules adapter \
  --load-pretrained-encoder-from /home4T_1/kzhong/study/s2t/output/multi/st/mustc_multilingual_st_transformer_m.pt \
  --load-pretrained-decoder-from /home4T_1/kzhong/study/s2t/output/multi/st/mustc_multilingual_st_transformer_m.pt

**bug:**
fairseq-train: error: unrecognized arguments: --lang-pairs en-de,en-es --homogeneous-batch --adapter-enc-dim 256 --adapter-enc-type per_lang --adapter-dec-dim 256 --adapter-dec-type per_lang --finetune-enc-modules adapter --finetune-dec-modules adapter --load-pretrained-decoder-from /home4T_1/kzhong/study/s2t/output/multi/st/mustc_multilingual_st_transformer_m.pt

**config:**
python:3.8
pytorch:
torch                  1.8.0+cu111
torchaudio             0.8.0
torchvision            0.9.0+cu111
fairseq: 0.10.2 installed by clone and pip install --editable ./ 

i am followed the example instruction:[https://github.com/formiel/fairseq/blob/master/examples/speech_to_text/docs/adapters.md](url), i think it is should be run.
